### PR TITLE
Update unit color properties

### PR DESCRIPTION
**Pull request by ron-murhammer**
Update unit color properties

### CODE REVIEW

File: map/map.properties [hunk 1]
```diff
@@ -14,15 +14,15 @@ color.Chinese=644664
...
-#units.color.<player> applies the given color's hue and saturation to the player's units
-#units.color.Russians=00BBBB
-#units.color.brightness.<player> allows the brightness (-100 to 100) to be adjusted when above unit color is set 
-#units.color.brightness.Russians=25
-#units.color.flip.<player> flips the player's units horizontally
-#units.color.flip.Russians=true
-#units.color.ignore provides a comma delimited list of units that aren't impacted by the above units.color properties
-#units.color.ignore=factory,fighter
+# Unit transform settings (apply effects similar to colorize functions of image editing tools)
+#units.transform.color.<player> applies the given color's hue and saturation to the player's units
```
[the remark this answers is not in the document]
Comment (ron-murhammer) [reply map/map.properties]: Uh not following this? I believe "player's units" is correct?

**Comment by ron-murhammer** [comment]
Follow up PR: https://github.com/triplea-maps/the_pact_of_steel/pull/34

@DanVanAtta For these types of minor changes, its probably easier for you to just make a follow up PR with the improvements and tag me since we don't really review PRs in this repo.

File: map/map.properties (hunk 1 above)
[the remark this answers is not in the document]
Comment (DanVanAtta) [reply map/map.properties]: Reviewed my grammar rules, I was wrong - you're originally right @ron-murhammer on the grammar here.

**Comment by DanVanAtta** [comment]
@ron-murhammer sounds good for a follow-up flow.

**Pull request merged**